lib/api_client_base/request.rb

```diff
@@ -60,7 +60,16 @@ def default_action
 
     def default_uri
       uri = if !api_client_base_path.nil?
-              URI.join(host, api_client_base_path)
+              path = URI.parse(host).path
+              if !path.empty?
```

Comment:
Ok great. The if statement two lines above isn't a great example, but to make this more readable you can avoid double negative and do:

```suggestion
              if path.present?
```

We gotta make sure that ActiveSupport's presence module is loaded into Ruby.

Reply:
Sure!